lib/membership/member.js

```diff
@@ -28,10 +28,39 @@ function Member(ringpop, update) {
     this.address = update.address;
     this.status = update.status;
     this.incarnationNumber = update.incarnationNumber;
+    this.dampScore = update.dampScore || ringpop.config.get('dampScoringInitial');
```

Comment:
yes, good catch. Since dampScores are not yet disseminated, this isn't an issue...yet. I'll fix no matter.